pom.xml

```diff
@@ -69,6 +69,18 @@
             <version>2.0.22-beta</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.jenkins-ci.plugins.workflow</groupId>
+            <artifactId>workflow-job</artifactId>
+            <version>1.4</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.jenkins-ci.plugins.workflow</groupId>
+            <artifactId>workflow-cps</artifactId>
+            <version>1.4</version>
+            <scope>test</scope>
+        </dependency>
```

Comment:
Why are you adding new `test`-scoped dependencies to a PR which does not add or modify tests?